cloud-controller-manager/do/loadbalancers.go

```diff
@@ -32,6 +32,10 @@ import (
 )
 
 const (
+	// annoDOLoadBalancerID is the annotation specifying the load-balancer ID
+	// used to enable fast retrievals of load-balancers from the API by UUID.
+	annoDOLoadBalancerID = "service.k8s.digitalocean.com/load-balancer-id"
```

Comment:
I think we should use `doks.digitalocean.com/load-balancer-id` here.

Reply:
Replaced by `kubernetes.digitalocean.com/load-balancer-id` as discussed via DM to make this less specific about DOKS. (DIY clusters will use the annotation as well.)